apix-rest-v1/src/main/java/eu/xenit/apix/rest/v1/nodes/NodesWebscript1.java

```diff
@@ -587,21 +587,42 @@ public void createNode(final CreateNodeOptions createNodeOptions, WebScriptRespo
                     @Override
                     public Object execute() throws Throwable {
                         NodeRef parent = new NodeRef(createNodeOptions.parent);
-                        NodeRef copyFrom = null;
+                        MetadataChanges metadataChanges;
```

Comment:
Can you move this down to where the constructor of MetadataChanges is called? It is better to move it closer to where it is actually used. That way when people want to use the variable they first don't have to look where it is initialized.